src/Filters/FiltersBelongsTo.php

```
@@ -0,0 +1,76 @@
+use Illuminate\Support\Arr;
+
+/**
+ * @template TModelClass of \Illuminate\Database\Eloquent\Model
+ *
+ * @template-implements Filter<TModelClass>
+ */
+class FiltersBelongsTo implements Filter
+{
+    public function __invoke(Builder $query, mixed $value, string $property): void
+    {
+        $values = array_values(Arr::wrap($value));
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Arr' in method '__invoke'. (undefined)

(StaticAccess)